src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3695,18 +3695,25 @@
      * <p>For example:</p>
      *
      * <pre>
-     *     ? super List
+     *     List&lt;? super List&gt; list;
      * </pre>
      *
      * <p>parses as:</p>
      *
      * <pre>
-     * +--TYPE_ARGUMENT
-     *     |
-     *     +--WILDCARD_TYPE (?)
-     *     +--TYPE_LOWER_BOUNDS
-     *         |
-     *         +--IDENT (List)
+     *--VARIABLE_DEF -&gt; VARIABLE_DEF
```

Comment:
after discussion with other core developers we decided that ast should start with node

so please remove preceding `--` characters and shift left other lines

smth like
```
     ...
     * VARIABLE_DEF -&gt; VARIABLE_DEF
     *  |--MODIFIERS -&gt; MODIFIERS
     ...
```